src/Sylius/Behat/Context/Setup/PromotionContext.php

```diff
@@ -170,4 +170,18 @@ public function itGivesOffEveryProductClassifiedAs(PromotionInterface $promotion
 
         $this->objectManager->flush();
     }
+
+    /**
+     * @Given /^([^"]+) gives ("(?:€|£|\$)[^"]+") off on every product (classified as "[^"]+")$/
+     */
+    public function itGivesFixedOffEveryProductClassifiedAs(PromotionInterface $promotion, $discount, TaxonInterface $taxon)
+    {
+        $action = $this->actionFactory->createItemFixedDiscount($discount);
+        $configuration = array_merge(['filters' => ['taxons' => [$taxon->getCode()]]], $action->getConfiguration());
+        $action->setConfiguration($configuration);
+
+        $promotion->addAction($action);
+
```

Comment:
You could extract some logic to a private method as this method and the previous one are virtually the same.